[DOCS] Adding metadata to articles (#18153)
openvinotoolkit/openvino: docs/Documentation/openvino_ecosystem.md

@sphinxdirective

.. meta::

>    :description: OpenVINO™ is an ecosystem of utilities such as Neural Network Compression Framework and OpenVINO Training Extensions that help develop deep learning solutions.

Author: How about?:

```suggestion
   :description: OpenVINO™ is an ecosystem of utilities that have advanced capabilities, which help develop deep learning solutions.
```
